### PR TITLE
Add NetworkZoneApi

### PR DESCRIPTION
**Pull request by denysoblohin-okta**
Internal ref: OKTA-453780  OKTA-453781

### CODE REVIEW

File: templates/openapi-generator/model/ObjectSerializer.mustache [hunk 1]
```diff
@@ -234,7 +234,10 @@ export class ObjectSerializer {
   /**
     * Parse data from a string according to the given media type
     */
-  public static parse(rawData: string, mediaType: string | undefined) {
+  public static parse(rawData: string, mediaType: string | undefined, dataType: string | null) {
+    if (dataType === 'void') {
```
Comment (oleksandrpravosudko-okta) [inline templates/openapi-generator/model/ObjectSerializer.mustache:238]: let's check with openapi-spec channel if the spec can be updated with `204` response

File: src/generated/models/ObjectSerializer.js [hunk 2]
```diff
@@ -1724,7 +1724,7 @@ class ObjectSerializer {
     static deserialize(data, type, format) {
         // polymorphism may change the actual type.
         type = ObjectSerializer.findCorrectType(data, type);
-        if (data == undefined) {
+        if (data === undefined || type === 'void') {
```
Comment (oleksandrpravosudko-okta) [inline src/generated/models/ObjectSerializer.js:1727]: should be changed in template as well?

Comment (denysoblohin-okta) [reply src/generated/models/ObjectSerializer.js]: Removed, unnecessary code

File: templates/openapi-generator/model/ObjectSerializer.mustache (hunk 1 above)
Comment (denysoblohin-okta) [reply templates/openapi-generator/model/ObjectSerializer.mustache:238]: https://github.com/okta/okta-oas3/pull/147

**Review by oleksandrpravosudko-okta — APPROVED** [review]

**Pull request merged**